tls/certificate.go

```diff
@@ -67,6 +67,12 @@ func (f FileOrContent) String() string {
 	return string(f)
 }
 
+// Path returns true if the FileOrContent is a file path, otherwise returns false
+func (f FileOrContent) Path() bool {
```

Comment:
if it's a boolean, `isPath` or you must return a string with filepath if you have one ?